curriculum/challenges/english/25-front-end-development/lecture-understanding-how-to-work-with-floats-and-positioning-in-css/672aa86da9937560d3dfe3d6.md

```diff
@@ -8,7 +8,56 @@ dashedName: what-are-common-use-cases-for-using-floats
 
 # --description--
 
-Watch the video lecture and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What are common use cases for using floats, and how do they work?
+
+Floats in CSS are a technique originally designed to allow text to wrap around an element, such as an image. Over time, however, developers found new ways to use floats, applying them to layout design in creative ways. While modern layout methods like Flexbox and Grid are now more commonly used, understanding floats is still important, especially when working with older code or needing to achieve specific layout effects.
+
+When an element is floated, it’s taken out of the normal document flow and pushed to the left or right of its container. The content that follows will wrap around the floated element, filling the remaining space. One classic use is wrapping text around images, where an image is floated to one side, and text wraps around it. This technique is still widely used, especially in articles and blogs where images need to be placed alongside text. Here is the code example:
+
+```html
+<body>
+  <div class="container">
+    <img src="https://via.placeholder.com/150" alt="Placeholder Image">
+    <p> This is an example of text flowing around a floated image.</p>
+  </div>
+</body>
+```
+
+```css
+.container {
+  border: 1px solid black;
+}
+
+img {
+  float: left;
+  margin-right: 20px;
+}
+
+In the above example, the container doesn’t wrap around the floated image. The image is out of the normal document flow, and the container collapsed to zero height because it doesn’t “see” the floated children elements. 
```

Comment:
```suggestion
In the above example, the container doesn't wrap around the floated image. The image is out of the normal document flow, and the container collapsed to zero height because it doesn't "see" the floated children elements. 
```